modules/core/src/main/java/org/apache/ignite/internal/processors/cache/GridCacheContextInfo.java

```diff
@@ -51,9 +55,10 @@ public class GridCacheContextInfo<K, V> {
      * Constructor of full cache context.
      *
      * @param cctx Cache context.
+     * @param kctx Kernel context.
      * @param clientCache Client cache or not.
      */
-    public GridCacheContextInfo(GridCacheContext<K, V> cctx, boolean clientCache) {
+    public GridCacheContextInfo(GridCacheContext<K, V> cctx, GridKernalContext kctx, boolean clientCache) {
```

Comment:
You don't need to pass KernalContext here.
It is available via  `cctx.kernalContext()`.